pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -338,7 +338,7 @@ func getMarkerScanners(logsCommandName, securityPolicyCommandFormat, setProbeCom
 		func(g osgraph.Graph, f osgraph.Namer) []osgraph.Marker {
 			return deployanalysis.FindDeploymentConfigReadinessWarnings(g, f, setProbeCommandName)
 		},
-		routeanalysis.FindMissingPortMapping,
+		routeanalysis.FindPortMappingIssues,
```

Comment:
Might be nice to have more smaller, more targeted methods, but it's a small nit.